### PR TITLE
📱 DEMOCRACY Deutschland (Project)

### PR DESCRIPTION
**Pull request by ulfgebhardt**
Add the `DEMOCRACY Deutschland` Project.

More Info:
- https://www.democracy-deutschland.de
- https://github.com/demokratie-live/
- https://github.com/demokratie-live/democracy-client/issues

Web-Version: https://democracy-app.de/
Android: https://play.google.com/store/apps/details?id=de.democracydeutschland.app
iOS: https://itunes.apple.com/us/app/democracy/id1341311162?l=de&ls=1&mt=8
APK: https://www.democracy-deutschland.de

### CODE REVIEW

File: _data/projects/DEMOCRACY.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- ios
+- react
+- reactnative
+- express
+- next
+- nextjs
+- GraphQL
+- MongoDB
+upforgrabs:
+  name: good first issue
```
Comment (ritwik12) [inline _data/projects/DEMOCRACY.yml]: @ulfgebhardt Please change this to "Is Good First Issue"

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/DEMOCRACY.yml (hunk 1 above)
Comment (ulfgebhardt) [reply _data/projects/DEMOCRACY.yml]: Done <3

**Comment by ulfgebhardt** [comment]
@ritwik12 is this now appropriate?

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@ulfgebhardt Thanks!!

**Pull request merged**